app/controllers/staff/users_controller.rb

```diff
@@ -17,6 +17,7 @@ def new
 
   def create
     @user = User.new(user_params)
+    @user.active = params[:user][:active]
```

Comment:
Wondering if we could make `user_params` deal with this as we do with the other params and we can continue to rely on strong parameters?

Reply:
Unfortunately brakeman objects to `active` as a param name for mass assignment (https://dxw.slack.com/archives/CN0H2L066/p1583249732015700) - I can change the name or tell brakeman to ignore the warning?